dev/io.openliberty.microprofile.openapi.4.1.internal_test/bnd.bnd

```diff
@@ -0,0 +1,33 @@
+#*******************************************************************************
+# Copyright (c) 2025 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+
+Bundle-Name: MicroProfile OpenAPI Unit tests
+Bundle-SymbolicName: io.openliberty.microprofile.openapi.4.1.internal_test
+Bundle-Description: MicroProfile OpenAPI 4.1 tests, version ${bVersion}
+
+bVersion=1.0
+
+Import-Package: \
+    *
+
+test.project: true
+
+-buildpath: \
+	io.openliberty.microprofile.openapi.4.0.internal.services;version=latest,\
+	io.openliberty.io.smallrye.openapi41.core;verison=latest,\
+	io.openliberty.org.eclipse.microprofile.openapi.2.0,\
```

Comment:
```suggestion
	io.openliberty.org.eclipse.microprofile.openapi.4.1,\
```
I assume this is right, but if it isn't, then don't do it.

Reply:
Yeah, I think this is right. Shouldn't affect the tests either way.

Reply:
So I was saying the change I was proposing was right, but if it isn't don't do the change.  You said it is right and didn't do the change.  So that confuses me.  Seems strange to use 4.1 impl with 2.0 API though.